web-ui/src/main/resources/catalog/components/ng-skos/templates/skos-concept.html

```diff
@@ -1,43 +1,40 @@
-<div class="skos-concept">
-  <div class="top top-alt">
-    <div class="btn-group pull-right">
-      <button
-        class="btn btn-default"
+<div title="{{'browseThesaurus' | translate}}">
+  <div data-gn-slide-toggle="true">{{'browseThesaurus' | translate}}</div>
+  <ol class="breadcrumb">
+    <li>
+      <a
+        class="btn btn-xs btn-link"
         title="{{'moveToTopConcept' | translate}}"
         ng-click="topConcept(concept.thesaurus)"
       >
-        <i class="fa fa-level-up"></i>
-      </button>
+        <i class="fa fa-home"></i>
+      </a>
+    </li>
+    <li ng-repeat="c in concept.previous.slice(1).reverse()">
+      <a
+        class="skos-concept-relation-previous"
+        ng-click="navigateConcept(c, true, $index)"
+      >
+        <span
+          role="menuitem"
+          tabindex="-1"
+          ng-if="prefLabel"
+          skos-label="c"
+          lang="{{language}}"
+        ></span>
+      </a>
+    </li>
+    <li class="active" data-ng-hide="concept.prefLabel['eng'] == 'topConcepts'">
```

Comment:
Is this ok to check for `eng` value? There are additional occurrences of this check in the file.

Reply:
Yes, see comment above.